plone/app/event/portlets/portlet_calendar.py

```diff
@@ -211,12 +211,9 @@ def cal_data(self):
                 sort_order=None,
             )
 
-            # restrict start/end with those from query, if given.
-            if "start" in query and query["start"] > start:
-                start = query["start"]
-            if "end" in query and query["end"] < end:
-                end = query["end"]
-
+            # When having a collection as context, we should use the configured query
+            # except the start and the end, which should be
+            # taken from the current calendar start and end
```

Comment:
Will the removal of the restriction still apply for a query for events that are _not_ in a collection as context?

Reply:
It never did. This code was before and still is inside a conditional that checks for the context being a collection.